src/openrct2-cli/Cli.cpp

```diff
@@ -21,7 +21,7 @@ int main(int argc, const char** argv)
 {
     int32_t rc = EXIT_SUCCESS;
     int runGame = cmdline_run(argv, argc);
-    core_init();
+    Platform::CoreInit();
```

Comment:
```suggestion
    Platform::Init();
```
I think this would be a bit nicer, just a suggestion.